### PR TITLE
h265dec: Fix vpsclient leak

### PR DESCRIPTION
**Pull request by dabrain34**

### CODE REVIEW

File: test/test.cpp [hunk 1]
```diff
@@ -144,9 +144,7 @@ class VideoParserClient : public VkParserVideoDecodeClient {
 
     bool UpdatePictureParameters(VkPictureParameters* params, VkSharedBaseObj<VkParserVideoRefCountBase>& shared, uint64_t count) final
     {
-        // fprintf(stdout, "%s: %" PRIu64 "\n", __FUNCTION__, count);
         fprintf(stdout, "[%lu] %s\n", syscall(SYS_gettid), __FUNCTION__);
-        shared = PictureParameterSet::create();
```
Comment (ceyusa) [inline test/test.cpp:149]: AFAIK, the shared object is created by the parser library and returned to the application which will use it to store custom data.

Comment (dabrain34) [reply test/test.cpp:149]: I was not sure if it was necessary indeed. My guess was the test application does not need this shared picture. Let me double check

Comment (dabrain34) [reply test/test.cpp]: Should be better now :)

**Comment by ceyusa** [comment]
Then, change the patch summary :)

**Pull request merged**